text/conversations/a_sagus/a2301_mapper.stringtable

```diff
@@ -386,7 +386,7 @@ Dann sieht er wieder dich an und sagt leise. „Ihr solltet auf Euren Reisen jed
       <ID>67</ID>
       <DefaultText>„Aber ich könnte Euch helfen, es zu finden. Für eine kleine Gegenleistung natürlich.“
 
-Er spricht leiser. „Habt Ihr schon vom Wandelnden Gott gehört? Retter von Sagus Klippen in früherer Zeit und so weiter? Man sagt, er hatte einst eine Zuflucht unter der Stadt. Ein Ort, an den er sich zurückzog, um seine Vergangenheit zu reflektieren und seine Zukunft zu planen. Möglichkeitskarten zu zeichnen. Diese Zuflucht ist der eine wichtige Ort, den ich in Sagus Klippen nie gesehen habe, und der in meiner Karte noch fehlt.“</DefaultText>
+Er spricht leiser. „Habt Ihr schon vom Wandelnden Gott gehört? Retter von Sagus Klippen in früherer Zeit und so weiter? Man sagt, er hatte einst eine Zuflucht unter der Stadt. Ein Ort, an den er sich zurückzog, um seine Vergangenheit zu reflektieren und seine Zukunft zu planen. Ihr wisst schon... Die Möglichkeit, eine Karte zu erstellen. Diese Zuflucht ist der eine wichtige Ort, den ich in Sagus Klippen nie gesehen habe, und der in meiner Karte noch fehlt.“</DefaultText>
```

Comment:
You know... making possibility maps.

Für ist das nicht eine Möglichkeit eine Karte zu erstellen, sondern eine Karte mit Möglichkeiten.

Reply:
Danke fürs Feedback. So besser?

```suggestion
Er spricht leiser. „Habt Ihr schon vom Wandelnden Gott gehört? Retter von Sagus Klippen in früherer Zeit und so weiter? Man sagt, er hatte einst eine Zuflucht unter der Stadt. Ein Ort, an den er sich zurückzog, um seine Vergangenheit zu reflektieren und seine Zukunft zu planen. Ihr wisst schon... Eine Karte mit Möglichkeiten. Diese Zuflucht ist der eine wichtige Ort, den ich in Sagus Klippen nie gesehen habe, und der in meiner Karte noch fehlt.“</DefaultText>
```